packages/web-runtime/src/services/clientRegistration.ts

```diff
@@ -29,7 +29,7 @@ async function post(url, data) {
 }
 
 export async function registerClient(openIdConfig) {
-  const clientData = sessionStorage.getItem('dynamicClientData')
+  const clientData = JSON.parse(sessionStorage.getItem('dynamicClientData'))
```

Comment:
I believe this was an actual bug, I have no idea how to test the whole feature, which is why I did not add a Changelog entry.